src/python/bot/fuzzers/ml/gradientfuzz/run_constants.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2019 Google LLC
```

Comment:
Let's hold off on landing the actual code until a later change. For now just include a skeleton of a task which logs something when it runs so that we can confirm it's working. I don't want this CL to get too large, and don't want any ambiguity when we test to see if the scheduling portion is working.

Reply:
+1

Reply:
Yup -- file removed until later commit!